views/tables/ConfigurationsTable/index.jsx

```diff
@@ -102,7 +107,33 @@ const ConfigurationsTable = (props) => {
             ]}
           />
         )
-
+      case 'status':
+        return configuration[property] === 1 ||
+          !Object.hasOwn(configuration, property) ? (
```

Comment:
this feels like logic that could go in a client-side configuration model to make it more obvious what this is doing. Something like `ConfigurationModel.isActive(configuration)`.